samples/vanilla/index.html

```diff
@@ -0,0 +1,282 @@
+            <ef-radio-button name="gender">Male</ef-radio-button>
+            <ef-radio-button name="gender">Female</ef-radio-button>
+            <ef-radio-button name="gender">Prefer not to say</ef-radio-button>
+          </div>
+        </div>
+        <div class="form-input">
+          <label>Date of Birth</label>
+          <ef-datetime-picker></ef-datetime-picker>
+        </div>
+        <div class="form-input">
+          <label>Address</label>
+          <textarea autoCorrect="off" autoComplete="off" autoCapitalize="off" rows="4" cols="50"></textarea>
+        </div>
+        <div class="form-input">
+          <label>Email *</label>
+          <ef-email-field
+            placeholder="user@refinitiv.com">
+          </ef-email-field>
+        </div>
+        <div class="form-input">
+          <label>Job Function</label>
+          <div id="job-panel" class="job-panel" role="button">
+            <div id="job-display"></div>
+            <ef-icon icon="arrow-down-fill"></ef-icon>
+          </div>
+          <ef-overlay-menu id="menu">
+            <ef-item for="saletrade">Sales & Trading</ef-item>
+            <ef-item for="maandcr">M&A and Capital Raising</ef-item>
+            <ef-item for="research">Research / Analysis</ef-item>
+            </ef-overlay-menu>
+          <ef-overlay-menu id="saletrade">
+            <ef-item value="Trader">Trader*</ef-item>
+            <ef-item value="Sales">Sales*</ef-item>
+            <ef-item value="Broker">Broker*</ef-item>
+          </ef-overlay-menu>
+          <ef-overlay-menu id="maandcr">
+            <ef-item value="ECM / DCM - Origination / Syndication">ECM / DCM - Origination / Syndication*</ef-item>
+            <ef-item value="M&A / Coverage Banker Senior">M&A / Coverage Banker Senior</ef-item>
+            <ef-item value="M&A / Coverage Banker Junior">M&A / Coverage Banker Junior</ef-item>
+            <ef-item value="Private Equity / Venture Capital">Private Equity / Venture Capital</ef-item>
+            <ef-item value="Investment Banking Business Manager">Investment Banking Business Manager</ef-item>
+          </ef-overlay-menu>
+          <ef-overlay-menu id="research">
+            <ef-item value="Research / Market Analyst">Research / Market Analyst*</ef-item>
+            <ef-item value="Quant">Quant</ef-item>
+            <ef-item value="Economist">Economist</ef-item>
+            <ef-item value="Investment Strategist">Investment Strategist</ef-item>
+          </ef-overlay-menu>
+        </div>
+        <div class="form-input">
+          <ef-checkbox>I want to receive news and updates via email</ef-checkbox> 
+        </div>
+        <div class="form-footer" slot="footer">
+          <ef-button id="confirm-button" cta>Confirm</ef-button>
+          <ef-button id="cancel-button">Cancel</ef-button>
+        </div>
+      </ef-dialog>
+    </div>
+    
+    <script>
+      const defaultTheme = document.documentElement.getAttribute('theme');
```

Comment:
Can we move these JS code to `index.js`? I think it easier to read and maintenance.